packages/jest-matchers/src/global_object.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
@thymikee that's a good name! i'll use that.

and yeah, our file naming conventions are broken :(